tools/runltp-ng/backend.pm

```diff
@@ -383,13 +384,21 @@ sub ssh_start
 	run_string($self, "$self->{'root_password'}");
 	sleep(1); #hack wait for prompt
 	wait_prompt($self);
+	if ($user ne 'root') {
+		run_string($self, "sudo su");
+		wait_prompt($self);
+	}
 }
 
 sub ssh_stop
 {
 	my ($self) = @_;
+	my $user = $self->{'ssh_user'}; 
 
 	run_string($self, "exit");
+	if ($user ne 'root') {
+		run_string($self, "exit");
+	}
 
```

Comment:
this could be written in a shorter way, i.e.

run_string($self, "exit") if ($user ne 'root');